sdk/build.gradle

```diff
@@ -1,234 +1,232 @@
+	}
+	integrationTest {
+		java.srcDir "$projectDir/src/integrationTest/java"
+
+		compileClasspath += sourceSets.main.output
+		runtimeClasspath += sourceSets.main.output
+	}
 }
 
 idea {
-    module {
-        sourceDirs -= file("src/integrationTest/java")
-        testSourceDirs += file("src/integrationTest/java")
-    }
+	module {
+		sourceDirs -= file("src/integrationTest/java")
+		testSourceDirs += file("src/integrationTest/java")
+	}
 }
 
 jacocoTestReport {
-    // make sure to use any/all test coverage data for the report
-    executionData fileTree(dir: buildDir, include: "jacoco/*.exec")
-
-    // remove generated proto files from report
-    afterEvaluate {
-        classDirectories.setFrom(files(classDirectories.files.collect {
-            fileTree(dir: it, exclude: '**/proto/**')
-        }))
-    }
-
-    // configure it so only xml is generated for the report
-    reports {
-        xml.enabled true
-        html.enabled true
-        csv.enabled false
-    }
-
-    // make sure we run all tests before this report is made
-    dependsOn tasks.withType(Test)
+	// make sure to use any/all test coverage data for the report
+	executionData fileTree(dir: buildDir, include: "jacoco/*.exec")
+
+	// remove generated proto files from report
+	afterEvaluate {
+		classDirectories.setFrom(files(classDirectories.files.collect {
+			fileTree(dir: it, exclude: '**/proto/**')
+		}))
+	}
+
+	// configure it so only xml is generated for the report
+	reports {
+		xml.enabled true
+		html.enabled true
+		csv.enabled false
+	}
+
+	// make sure we run all tests before this report is made
+	dependsOn tasks.withType(Test)
 }
 
 task integrationTest(type: Test) {
-    testClassesDirs = sourceSets.integrationTest.output.classesDirs
-    classpath = sourceSets.integrationTest.runtimeClasspath
+	testClassesDirs = sourceSets.integrationTest.output.classesDirs
+	classpath = sourceSets.integrationTest.runtimeClasspath
 
-    onlyIf {
-        System.getenv('OPERATOR_ID') != null && System.getenv('OPERATOR_KEY') != null || System.getenv('CONFIG_FILE') != null
-    }
+	onlyIf {
+		System.getenv('OPERATOR_ID') != null && System.getenv('OPERATOR_KEY') != null || System.getenv('CONFIG_FILE') != null
+	}
 }
 
 tasks.withType(Javadoc) {
-    source = sourceSets.main.allJava + file("$buildDir/generated/sources/annotationProcessor/java/main")
+	source = sourceSets.main.allJava + file("$buildDir/generated/sources/annotationProcessor/java/main")
 }
 
 tasks.withType(Test) {
-    useJUnitPlatform()
+	useJUnitPlatform()
 
-    // NOTE: Uncomment to enable trace logs in the SDK during tests
-    // jvmArgs "-Dorg.slf4j.simpleLogger.log.com.hedera.hashgraph=trace"
+	// NOTE: Uncomment to enable trace logs in the SDK during tests
+	// jvmArgs "-Dorg.slf4j.simpleLogger.log.com.hedera.hashgraph=trace"
 
-    // this task will fail on the first failed test
-    failFast = true
+	// this task will fail on the first failed test
+	failFast = true
 
-    finalizedBy jacocoTestReport
```

Comment:
That's fine, but that was to fix code coverage not being reported. Can you look into seeing why the coverage report is not being generated then?

Reply:
Or at least create a ticket so it can be worked on? Capturing code coverage is important so we can continuously improve.

Reply:
Created issue for this: https://github.com/hashgraph/hedera-sdk-java/issues/639 If it comes down to it we might want to revert this change and simply do `./gradlew compileJava compileTestJava compileIntegrationTestJava` instead of ./gradlew build`